### DISCUSSION TITLE
Regarding Disabling 608 and 708 Closed Captions

### DISCUSSION THREAD

**Discussion by bbddstory** [videojs/video.js#8131] [General]
Hi people,

I understand there's an boolean option called `parse708captions` for disabling the parsing of 708 captions. Just wondering why isn't there one for disabling 608 captions? I'm asking because we've been getting a lot of requests from customers wanting to disable 608 all together due to its many limitations comparing to 708. Was there any reason for this or simply because 608 is older? Thank you.

Reference: [parse708captions](https://github.com/videojs/http-streaming#parse708captions)

Leon

**Comment by gkatsev** [comment]
608 is simpler and more common. We've had issues where a stream had properly set up 608 captions and also had 708 captions, but the 708 captions weren't set up properly and produced junk.

I'd generally expect the stream to provide only one or the other and not both at the same time.

**Reply by bbddstory** [reply]
Thanks for the explanation @gkatsev, much appreciated. Cheers.
